spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/security/oauth2/resource/servlet/OAuth2ResourceServerAutoConfigurationTests.java

```diff
@@ -745,4 +793,15 @@ SecurityFilterChain testSecurityFilterChain(HttpSecurity http) throws Exception
 
 	}
 
+	@Configuration(proxyBeanMethods = false)
+	@EnableWebSecurity
```

Comment:
I don't think `@EnableWebSecurity` is needed here.

Reply:
Removed